templates/stats/index.html

```diff
@@ -61,97 +61,130 @@ <h2 v-if="selected == 'Antal cashflows'">Flest Cashflows</h2>
-                <optgroup label="Nämnder">
-                  <option v-for="committee in activeCommittees().filter(c => c.type == 'committee')"
-                          :key="committee.id" v-text="committee.name" v-bind:value="committee.id"></option>
-                </optgroup>
-                <optgroup label="Projekt">
-                  <option v-for="committee in activeCommittees().filter(c => c.type == 'project')"
-                          :key="committee.id" v-text="committee.name" v-bind:value="committee.id"></option>
-                </optgroup>
-                <optgroup label="Övrigt">
-                  <option v-for="committee in activeCommittees().filter(c => c.type != 'committee' && c.type != 'project')"
-                          :key="committee.id" v-text="committee.name" v-bind:value="committee.id"></option>
-                </optgroup>
-              </select>
-            </div>
-          </div>
-        </td>
-        <td>
-          <div class="input">
-            <div class="select" v-if="selectedCommittee.committeeId > 0">
-              <select name="costCentre[]" v-model="selectedCommittee.costCentreId" required>
-                <option value="0">- Välj -</option>
-                <option v-for="costCentre in activeCostCentres()" v-text="costCentre.name" v-bind:value="costCentre.id"></option>
+      <tbody>
+        <tr>
+          <!-- First Column for the Year Dropdown -->
+          <td>
+            <!-- Year Dropdown -->
+            <div class="select" style="width: 100%;">
+              <select 
+                v-model="selectedYear" 
+                @change="showDropdownN"
+                style="width: 100%;" 
+                :disabled="expense_parts.length > 1"
+              >
+                <option value="">- Välj år -</option>
+                <option v-for="year in years" :key="year" :value="year">
+                  {% verbatim %}
+                  {{ year }}
+                  {% endverbatim %}
+                </option>
               </select>
             </div>
-            <div class="select disabled" v-else>
-              <select name="costCentre[]" required disabled>
-                <option value="0">- Välj resultatställe först -</option>
+          </td>
+  
+          <!-- Second Column for the Cost Centre Dropdown -->
+          <td>
+            <!-- Cost Centre Dropdown (only visible if a year is selected) -->
+            <div class="select" style="width: 100%;">
+              <select 
+                name="costCentres[]" 
+                v-model="expense_parts[0].costCentre" 
+                required 
+                @change="costCentreHandler()"
+                style="width: 100%;"
+              >
+                <option value="">- Välj -</option>
+                <option
+                  v-for="(costCentre, index) in costCentres"
+                  :key="index"
+                  :value="costCentre"
+                >
+                {% verbatim %}
```

Comment:
This works, but there is also the `v-text` property

Reply:
Didn't know, thank you !